records/board/mlstl476qe.yaml

```diff
@@ -0,0 +1,11 @@
+common:
+    macros:
+        - IO_CONFIG_OVERRIDE
```

Comment:
Sorry, What doese the "default IO confirguratoin" mean, since the target is new one of STM32L476XX serials.
Do you mean We can use some available lpc11u35_xxx_if as the firmware to work with(debugger/programmer)

Best Regards,
Xinbai